src/javaharness/java/arcs/api/PECInnerPortImpl.java

```diff
@@ -155,6 +160,60 @@ public void SynchronizeProxy(StorageProxy storageProxy, Consumer<PortableJson> c
         postMessage(message);
     }
 
+    @Override
+    public void HandleStore(StorageProxy storageProxy, Consumer<PortableJson> callback, PortableJson data, String particleId) {
+        PortableJson message = jsonParser.parse("{}");
```

Comment:
Maybe create a helper for this and use it everywhere? Or, add a "createEmptyMessage" method on jsonParser itself?

Reply:
done (added emptyJson to parser)